### ISSUE TITLE
Lint: indentation for function parameters

### ISSUE THREAD

**Issue by relikd** [astral-sh/ruff#26799]
### Summary

It feels wrong if the linter does not complain about this code (even with ALL rules):

```py
def foo():
    if not some_func(dict_file, xml=xml_file, plist=plist_file,
compress=True,
    force=False):
        exit(1)
```

I am seraching for a config option to enforce multi-line function parameter alignment.

My previous linter would require that all lines have the same indentation as the first parameter.

```py
# either
def foo():
    if not some_func(dict_file, xml=xml_file, plist=plist_file,
                     compress=True,
                     force=False):
        exit(1)
# or
def foo():
    if not some_func(
            dict_file, xml=xml_file, plist=plist_file,
            compress=True,
            force=False):
        exit(1)
# or
def foo():
    if not some_func(
        dict_file, xml=xml_file, plist=plist_file,
        compress=True,
        force=False
    ):
        exit(1)
```

### Version

ruff 0.15.21 (3e1f63645 2026-07-09)

**Comment by ntBre** [comment]
I think this is a duplicate of https://github.com/astral-sh/ruff/issues/2402 because pycodestyle emits diagnostics on lines 3 and 4 here:

```console
❯ uvx pycodestyle try.py --show-source
try.py:3:1: E128 continuation line under-indented for visual indent
compress=True,
^
try.py:4:5: E128 continuation line under-indented for visual indent
    force=False):
    ^
```

I'd typically expect our formatter to handle this rather than the linter, but that issue tracks a lint rule that would handle this. 

The formatter currently produces:

```py
def foo():
    if not some_func(
        dict_file, xml=xml_file, plist=plist_file, compress=True, force=False
    ):
        exit(1)
```

without a trailing comma, or:

```py
def foo():
    if not some_func(
        dict_file,
        xml=xml_file,
        plist=plist_file,
        compress=True,
        force=False,
    ):
        exit(1)
```

with a trailing comma.

**Comment by relikd** [comment]
Perfect. But from what I see, this is not implemented yet? Any estimate when that could be? For the record, I dont use the formatter at all. I just want the linter to notify me of potential issues.

**Comment by ntBre** [comment]
I'm not sure on a timeline. There's actually an open implementation [PR](https://github.com/astral-sh/ruff/pull/13585) but still some open design questions from my read of the thread.
